libs/domain/picking/offline/feature.ts

```diff
@@ -31,7 +31,7 @@ export class OfflinePickingFeature implements AppFeature {
         PickingSyncActionHandlerService
       ),
       { provide: PickingListService, useClass: PickingListDefaultService },
-      { provide: PickingListAdapter, useClass: PickingListOfflineAdapter },
+      { provide: PickingListAdapter, useClass: PickingListDefaultAdapter },
```

Comment:
Why did you change this? In offline mode an offline adapter should be used instead of the default one.

Reply:
We should also change `DemoDataPlugin` to load picking lists from the online adapter and store it in the DB instead of the hardcoded data.
It will eliminate the need for this change.